draft-westerlund-tsvwg-sctp-dtls-chunk.md

```diff
@@ -173,8 +173,8 @@ to SCTP and the Upper Layer Protocol (ULP) using DTLS 1.3 for key management.
 +-------+-------+ +-----------------------------+-+
         ^                          ^            |
         |                          |            |
-        +--+-----------------------+            | keys
-      PPID |                                    |
```

Comment:
I would prefer `keys` here, since this is what is sent down.

Reply:
Whenever I put some text next to an arrow, the text describes what is sent. Like PPID, ContentType of keys. So please fix the text instead of the figure.